### ISSUE TITLE
Could I use my own datasets?

### ISSUE THREAD

**Issue by beathahahaha** [multimodal-art-projection/AutoKaggle#5]
Could I use my own datasets made by self, instead of using datasets on kaggle? In fact, I want to upload my own datasets and define a target by self, and give it a description. Could I make this?

**Comment by batuhanmtl** [comment]
> Could I use my own datasets made by self, instead of using datasets on kaggle? In fact, I want to upload my own datasets and define a target by self, and give it a description. Could I make this?

yes it's possible to do this. If you provide the necessary train.csv, test.csv, sample_submission.csv and overview.txt you can use the framework. I did what you said even with a different LLM integration and a different dataset.

**Comment by beathahahaha** [comment]
> > Could I use my own datasets made by self, instead of using datasets on kaggle? In fact, I want to upload my own datasets and define a target by self, and give it a description. Could I make this?
> 
> yes it's possible to do this. If you provide the necessary train.csv, test.csv, sample_submission.csv and overview.txt you can use the framework. I did what you said even with a different LLM integration and a different dataset.

thank you~I will try it later.
